pydgraph/util.py

```diff
@@ -52,5 +52,9 @@ def is_retriable_error(error):
 
 def is_connection_error(error):
     """Returns true if the error is caused connection issues."""
+    if isinstance(error, grpc._channel._InactiveRpcError):
```

Comment:
Can you add a comment explaining the two different types of error msgs that we are checkign for?